domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -830,6 +793,53 @@ class TopicListController @Inject constructor(
       .setChapterPlayState(nextChapterProgress?.chapterPlayState ?: ChapterPlayState.NOT_STARTED)
       .build()
   }
+
+  // TODO(#5344): Remove this in favor of per-classroom data handling.
```

Comment:
Mention the temporary setup here & the limitations in expecting only a single classroom currently.